beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -24,13 +23,10 @@ var processPendingBlocksPeriod = slotutil.DivideSlotBy(3 /* times per slot */)
 // processes pending blocks queue on every processPendingBlocksPeriod
 func (s *Service) processPendingBlocksQueue() {
 	ctx := context.Background()
-	locker := new(sync.Mutex)
```

Comment:
same applies here

Reply:
Thanks for elaborating - will revert, and add comment, so that the intent is clear.